loctool.js

```diff
@@ -507,6 +539,23 @@ try {
 
         fs.writeFileSync(target.getPath(), target.serialize(), "utf-8");
         break;
+
+    case "generate":
+        var project = ProjectFactory.newProject(settings, settings);
```

Comment:
Put all this in a generateMode.js file

Reply:
I've changed all to `generateMode` from `generateResource`

Reply:
I mean to put lines 568-580 in GenerateMode.js and export it as a function. That way, we can call it from here, and also call it from the unit tests.

Reply:
Due to misunderstanding, I've already changed the file to GenerateMode.js so I newly write the file as GenerateModeProcess.js to work on it. Please check the latest change again.